source/game_videos.md

```diff
@@ -0,0 +1,97 @@
+| RoboCup 2024 | 2024‑07‑20 14:15 | A | 1 | 2 | Berlin United | rUNSWift | [video](https://logs.naoth.de/2024-07-15_RC24-Videos/Field-A-NAOCAM_10_2/GX020206.MP4) | [GC](https://b-human.informatik.uni-bremen.de/public/logs/2024/RoboCup/gc/FieldA/log_2024-07-20_12-09-03_Berlin-United_rUNSWift.yaml) | [TCM](https://b-human.informatik.uni-bremen.de/public/logs/2024/RoboCup/tcm/FieldA/teamcomm_2024-07-20_14-15-22-664_Berlin%20United_rUNSWift_1stHalf.log) |
+| RoboCup 2024 | 2024‑07‑20 14:15 | A | 1 | 1 | Berlin United | rUNSWift | [video](https://logs.naoth.de/2024-07-15_RC24-Videos/Field-A-NAOCAM_10_2/GX010206.MP4) | [GC](https://b-human.informatik.uni-bremen.de/public/logs/2024/RoboCup/gc/FieldA/log_2024-07-20_12-09-03_Berlin-United_rUNSWift.yaml) | [TCM](https://b-human.informatik.uni-bremen.de/public/logs/2024/RoboCup/tcm/FieldA/teamcomm_2024-07-20_14-15-22-664_Berlin%20United_rUNSWift_1stHalf.log) |
+| RoboCup 2024 | 2024‑07‑20 09:15 | E | 2 | 2 | Nao Devils | rUNSWift | [video](https://logs.naoth.de/2024-07-15_RC24-Videos/Field-E-NAOCAM_10_/GX011234.MP4) | [GC](https://b-human.informatik.uni-bremen.de/public/logs/2024/RoboCup/gc/FieldE/log_2024-07-20_07-13-01_Nao-Devils_rUNSWift.yaml) | [TCM](https://b-human.informatik.uni-bremen.de/public/logs/2024/RoboCup/tcm/FieldE/teamcomm_2024-07-20_09-53-36-910_rUNSWift_Nao%20Devils_2ndHalf.log) |
+| RoboCup 2024 | 2024‑07‑20 09:15 | E | 2 | 1 | Nao Devils | rUNSWift | [video](https://logs.naoth.de/2024-07-15_RC24-Videos/Field-E-NAOCAM_10_/GX011233.MP4) | [GC](https://b-human.informatik.uni-bremen.de/public/logs/2024/RoboCup/gc/FieldE/log_2024-07-20_07-13-01_Nao-Devils_rUNSWift.yaml) | [TCM](https://b-human.informatik.uni-bremen.de/public/logs/2024/RoboCup/tcm/FieldE/teamcomm_2024-07-20_09-44-10-410_rUNSWift_Nao%20Devils_2ndHalf.log) |
+| RoboCup 2024 | 2024‑07‑20 09:15 | E | 1 | 1 | Nao Devils | rUNSWift | [video](https://logs.naoth.de/2024-07-15_RC24-Videos/Field-E-NAOCAM_10_/GX011232.MP4) | [GC](https://b-human.informatik.uni-bremen.de/public/logs/2024/RoboCup/gc/FieldE/log_2024-07-20_07-13-01_Nao-Devils_rUNSWift.yaml) | [TCM](https://b-human.informatik.uni-bremen.de/public/logs/2024/RoboCup/tcm/FieldE/teamcomm_2024-07-20_09-15-11-409_rUNSWift_Nao%20Devils_1stHalf.log) |
+| RoboCup 2024 | 2024‑07‑19 15:30 | C | 2 | 1 | HULKs | rUNSWift | | [GC](https://b-human.informatik.uni-bremen.de/public/logs/2024/RoboCup/gc/FieldC/log_2024-07-19_13-24-11_HULKs_rUNSWift.yaml) | [TCM]() |
+| RoboCup 2024 | 2024‑07‑19 15:30 | C | 1 | 1 | HULKs | rUNSWift | | [GC](https://b-human.informatik.uni-bremen.de/public/logs/2024/RoboCup/gc/FieldC/log_2024-07-19_13-24-11_HULKs_rUNSWift.yaml) | [TCM]() |
```

Comment:
The two lines above generate warnings, I think it's because there are no video links. Should be ok.